### ISSUE TITLE
## AXE a11y errors violations

### ISSUE THREAD

**Issue by qunabu** [EscolaLMS/Components#276]
## AXE a11y errors violations
    
1 accessibility violations were detected

| id             | impact  | description                                                                                                     | nodes |
| -------------- | ------- | --------------------------------------------------------------------------------------------------------------- | ----- |
| color-contrast | serious | Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds | 1     |
    

_Originally posted by @github-actions[bot] in https://github.com/EscolaLMS/Components/issues/274#issuecomment-1512561004_
